README.md

```diff
@@ -61,23 +61,31 @@ console.log(decoded)
 
 - `opts: Object`, optional
   - `poolSize: 10 * 1024`: Buffer pool size to allocate up front
+  - `minPoolSize: 147`: The minimum size the pool can be before it is re-allocated. Note this is important 
```

Comment:
> Note this is important

This ties directly to ones ability to read the length of the data correct? In `decode.js` the comment signifies this is equivalent to the number of bytes needed to read `Varint.encode(Number.MAX_VALUE).length` (147), Number.MAX_VALUE is a float though. Should it be `Varint.encode(Number.MAX_SAFE_INTEGER).length` (8)?

Reply:
When allocating a buffer the docs say ["<integer>"](https://nodejs.org/dist/latest/docs/api/buffer.html#buffer_class_method_buffer_alloc_size_fill_encoding) so I assume then the biggest you can allocate would be `Number.MAX_SAFE_INTEGER` meaning yes, you're right this should be 8 bytes, not 147. Good catch.